### ISSUE TITLE
Write message to bus

### ISSUE THREAD

**Issue by macdudeuk** [dturing/node-gstreamer-superficial#15]
Hi guys. If anyone feels like tackling it, it would be great to have a way to send a message to the GStreamer bus. In particular sending EOS before tearing down a pipeline.In fact a way to cleanly close a pipeline in general. If you are recording anything to a file this is pretty much mandatory and would make this module useful in more situations. Sadly such an enhancement is past my capabilities...

PS Love it so far.

**Comment by macdudeuk** [comment]
Actually I have amazed myself and have made and tested the changes to add pipeline.sendEOS(). Turned out to be trivial. I have done it outside of GIT (which I have no idea how to use) so have attached the two changed files. Sorry for being so lame!

[Pipeline.cpp.txt](https://github.com/dturing/node-gstreamer-superficial/files/913624/Pipeline.cpp.txt)
[Pipeline.h.txt](https://github.com/dturing/node-gstreamer-superficial/files/913625/Pipeline.h.txt)
